GoogleChromeLabs/ps-analysis-tool
# Feature: Interest groups bubble plotting

## Description
In this PR, we aim to create a container for interest group bubbles. So that it looks better when there are more number of interest group bubbles.

## Relevant Technical Choices
- A new module `bubbles` has been added. This module will manage the animation for the loading the interest group and joining interest group.
- this will also manage the expansion and minification of the interest group bubble container.

## Testing Instructions
- Clone the branch.
- In the terminal run `cd ee && npm i && npm run start`.
- Now open https://localhost:8080/index.html.
- Once interest group is added by advertiser you can click on this and expand the circle to see interest groups.

## Additional Information:



## Screenshot/Screencast

![Screenshot 2024-11-04 at 16 34 55](https://github.com/user-attachments/assets/bf261f51-8d74-4547-bd05-4f8f08209f98)

https://github.com/user-attachments/assets/2fe99e76-2c28-4cb6-8bd2-84c7507bccb2


---

## Checklist






Partially addresses #857
